src/main/java/no/uio/ifi/ltp/controllers/rest/ProxyController.java

```diff
@@ -158,4 +158,16 @@ protected String getElixirAAIToken(String bearerAuthorization) {
         return bearerAuthorization.replace("Bearer ", "");
     }
 
+    /**
+     * Gets TSD token.
+     *
+     * @param bearerAuthorization Elixir AAI token.
+     * @return TSD token.
+     */
+    @GetMapping("/gettoken")
```

Comment:
```suggestion
    @GetMapping("/tsd/token")
```

Reply:
We (Parisa and me) devcided it's best to do the change of the end-point and the lega-commander client in sync. So we approve this PR without doing the end-point name change, and create a new card in our Project board to solve these to required changed (lc and proxy) in sync.